articles/appliance/index.html

```diff
@@ -0,0 +1,137 @@
+</ul>
+
+<h2>Appliance Administration</h2>
+
+<!--
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/"></a>
+    <p>
+      
+    </p>
+  </li>
+-->
+
+<ul class="topic-links">
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="https://auth0.com/changelog/appliance">Appliance Release Notes</a>
+    <p>
+      This document shows the latest updates to the Auth0 Appliance.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/admin">Appliance Administrator's Manual</a>
+    <p>
+      This document covers factors Auth0 Appliance administrators should be aware of when working with production Appliance instances.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/dashboard">Appliance Dashboard/Configuration Area</a>
+    <p>
+      Regardless of whether you choose to use Auth0's cloud services or to run Auth0 through Appliance instances, the Management Dashboard is the interface through which you may configure things like your Applications, Connections, Users, and Rules.
+    </p>
+    <ul>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/appliance/custom-domains">Custom Domains</a>
+      </li>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/appliance/clock">Time Synchronization in the Appliance</a>
+      </li>
+    </ul>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/monitoring">Tools for Monitoring the Appliance</a>
+    <p>
+      In addition to providing tools for monitoring your Appliance instances, Auth0 provides integration with select third-party utilities.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/cli">Appliance Command-Line Interface Tools</a>
+    <p>
+      The Auth0 Appliance Command Line Interface (CLI) allows you to perform operations on your Appliance via authorized workstations.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/admin/creating-tenants">Automatic Creation of Tenants</a>
+    <p>
+      If your business needs require you to create tenants regularly, you may automate this process in your Appliance instances. For example, you might need to create one tenant for each customer or project that goes live.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/modules">Node.js Modules Available in Rules and Custom Database Connections for Appliances</a>
+    <p>
+      For security reasons, rules and custom database connections for Auth0 appliances run in a JavaScript sandbox. You can use the full power of the ECMAScript 5 language and a few selected libraries.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/webtasks">Webtasks</a>
+    <p>
+      Beginning with Build 7247, you may use the Appliance's version of  [Webtasks](http://webtask.io/) to execute your rules and custom database logic.
```

Comment:
@nkohari The markdown URL needs fixing, it does not render correctly. 

![image](https://cloud.githubusercontent.com/assets/11715799/18555711/7e9e2afc-7b71-11e6-849e-07eb184ee937.png)